src/sentry/static/sentry/app/views/organizationStats/projectTable.jsx

```diff
@@ -54,6 +54,10 @@ const ProjectTable = React.createClass({
           {projectTotals.map(item => {
             let project = projectMap[item.id];
 
+            if (!project) {
```

Comment:
This is not ideal, `projectTotals` has full list of stats grouped by project, and `projectMap` is a paginated list of projects. Rendered table is not guaranteed to be sorted.